### ISSUE TITLE
Negative numbers in int positional arguments

### ISSUE THREAD

**Issue by 011010101101** [alecthomas/kingpin#21]
Love the library, man. Thank you for your contribution!

You might wanna add getopt-like double dash (`--`) behavior so that negative numbers can be used in positional arguments. I tried to do that today in 1.3.2, but it assumes I'm trying to specify a flag. My example had a string positional argument and an Int64 positional argument, like so:

```
% cmd "my string" -123
error: unexpected arguments '-1 -2 -3', try --help
```

I was hoping to get `-123` as an Int64.

With getopt, I think you can do this and have it work correctly:

```
% cmd -- "my string" -123
```

Basically the `--` by itself means "everything after this is not a flag".
